keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -589,6 +604,10 @@ details). For now, we leave it here.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+ This feature will not work if the API server or etcd is unavailable as the controller-manager won't be even able get events or updates for StatefulSets. 
```

Comment:
Does this feature have a dependency on events? They are lossy, that doesn't seem right.

Reply:
None. I meant the update/create events not the events that individual components create.